packages/clerk-js/src/core/resources/Base.ts

```diff
@@ -47,6 +47,11 @@ export abstract class BaseResource {
 
     const { payload, status, statusText } = fapiResponse;
 
+    if (payload?.meta) {
+      const country = payload?.meta.responseHeaders?.country;
```

Comment:
```suggestion
      const country = payload.meta.responseHeaders?.country;
```
🙃 payload.meta exists inside `if`